node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/FileFinder.java

```diff
@@ -110,17 +110,19 @@ public FileFinder maxDepth(int maxDepth) {
      * @return true iff anything was matched and deleted
      */
     public boolean deleteRecursively(TaskContext context) {
+        final int limit = 20;
```

Comment:
Maybe worth a comment saying why there is a limit?

Reply:
Renamed the variable to `maxNumberOfDeletedPathsToLog`